tests/commands/test_command_ingest.py

```diff
@@ -68,4 +69,54 @@ def test_run(dataset, runner):
 
             if table_key != "image":
                 assert df.groupby(["TableNumber", "ImageNumber"]).size().sum() == \
-                       dataset["ingest"]["{}_nrows".format(table_name)]
\ No newline at end of file
+                       dataset["ingest"]["{}_nrows".format(table_name)]
+
+
+def test_run_defaults(dataset, runner):
+    opts = ["ingest"]
+
+    if dataset["munge"]:
+        opts += ["--munge"]
+    else:
+        opts += ["--no-munge"]
+
+    opts += [dataset["data_dir"]]
+
+    with backports.tempfile.TemporaryDirectory() as temp_dir:
+        sqlite_file = os.path.join(temp_dir, "test.db")
+
+        opts += ["sqlite:///{}".format(sqlite_file)]
+
+        result = runner.invoke(cytominer_database.command.command, opts)
+
+        assert result.exit_code == 0
+
+        config = configparser.ConfigParser()
+
+        config_file = pkg_resources.resource_filename(
+            "cytominer_database",
+            os.path.join("config", "config_htqc.ini")
```

Comment:
This config file works only for the `htqc` fixture
For `cellpainting`, it is `config_cellpainting.ini`

I just noticed that all the tests are being performed only on the `htqc`, not `cellpainting`. 

We'd need to look up

Reply:
@shntnu I've submitted #81 which runs the tests on both the `htqc` and `cellpainting` datasets.

Is this configuration file the correct default configuration file for the command line? If not, which configuration file should be used? Or, is the `--config` flag meant to work like the `--dataset` flag?

Reply:
Ah! I didn't realize you are testing defaults here. Apparently `test_run_defaults` was not obvious enough for me 😆 
The default config file should be `config_cellpainting.ini`. That means this test should be tested only on `cellpainting` and not `htqc`. But I don't know how you'd configure that 
@mcquin

Reply:
My guess is you'd have `@pytest.mark.skipif(...)` before `def test_run_defaults`, where `...` is a check for `dataset == htqc`

Reply:
Unfortunately, `dataset` isn't available in the context of `skipif` (it'd be really nice if it were). I've split the datasets out into their own fixtures for finer control.